training/deepspeech_training/util/signal_augmentations.py

```diff
@@ -0,0 +1,200 @@
+
+import os
+import re
+import math
+import random
+import numpy as np
+
+from multiprocessing import Queue, Process
+from .audio import gain_db_to_ratio, max_dbfs, normalize_audio, AUDIO_TYPE_NP, AUDIO_TYPE_PCM, AUDIO_TYPE_OPUS
+from .helpers import int_range, float_range, pick_value_from_range, MEGABYTE
+
+SPEC_PARSER = re.compile(r'^([a-z]+)(\[(.*)\])?$')
```

Comment:
Name the groups for better readability:
```suggestion
SPEC_PARSER = re.compile(r'^(?P<cls>[a-z]+)(\[(?P<params>.*)\])?$')
```
and then:
```python
match.group('cls')
```